### PR TITLE
fixed saved search query

### LINKED ISSUE #27444: bug: documentation on saved searches may be out of date
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/search-github/github-code-search/using-github-code-search#creating-and-managing-saved-searches

### What part(s) of the article would you like to see updated?

Currently, step 4 (more on this a little later 😄) reads as follows:

> If you click again on the search bar, you can now see your saved search in the "Saved searches" section under the search bar. Clicking on a saved search entry will add the query to the search bar and filter the suggestions accordingly.
To edit a saved search, in the "Saved searches" section, click to the right of the saved search.
To delete a saved search, click to the right of the saved search.

This does not match the recently updated UI. When I first focus on the search field, the saved search is listed as a suggestion by default.

Screenshot that shows the saved search as an autocomplete suggestion in the suggestions list.

When I type `saved:` the UI updates to only show my saved searches and surfaces the "Manage saved searches" option.

Screenshot demonstrating the updated UI with an option to manage saved searches.

It does not show the edit or delete options as mentioned in the docs. To access these, users must select the option to manage their saved searches.

Screenshot of the manage saved searches modal. This list the users saved searches, and for each entry, it shows options to edit or delete a saved search.

> Related: I opened this discussion because a saved search I have shows up under saved searches, but selecting the saved search triggers an error. https://github.com/orgs/community/discussions/63708

## Sidenote

Concerning step 4. Steps one to three are concerned with creating a saved search, and step four is concerned with managing an existing saved search. These feel like separate workflows and are not part of the same sequential steps. It will become clearer with the suggested updates above, so I recommend separating them into two separate ordered lists.

> NOTE: Please let me know if I should separate some of the above into separate issues. Thanks! I am also happy to open a pull request to address the documentation updates mentioned.

### Additional information

[maintainer edit]
Anyone can work on this by following the content plan [here](https://github.com/github/docs/issues/27444#issuecomment-1691696133)

### PR DESCRIPTION
**Pull request by npiyush97**
### Why:

Closes: #27444 

### Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by npiyush97** [comment]
@cmwilson21 created a new pull request and removed tags and images up to spec. Have a look

**Comment by cmwilson21** [comment]
@npiyush97 Thanks so much for updating the tags and images ✨ 

I'll leave it up to the reviewer to decide if the extra images are necessary since the [content plan](https://github.com/github/docs/issues/27444#issuecomment-1691696133) didn't call for them, but we really appreciate you taking the time to make these changes and looking over our screenshot guidance 💖 

I'll get this triaged for review! :zap:

**Comment by npiyush97** [comment]
> @npiyush97 Thanks so much for updating the tags and images ✨
> 
> I'll leave it up to the reviewer to decide if the extra images are necessary since the [content plan](https://github.com/github/docs/issues/27444#issuecomment-1691696133) didn't call for them, but we really appreciate you taking the time to make these changes and looking over our screenshot guidance 💖
> 
> I'll get this triaged for review! ⚡

Thought visual example/steps help understand better so added them.

**Comment by cmwilson21** [comment]
@npiyush97 And we appreciate that thoughtful thinking 💖

**Comment by npiyush97** [comment]
@cmwilson21 update on pr merge ?

**Comment by cmwilson21** [comment]
@npiyush97 Thanks for checking in. This is on the board waiting for review. Someone will have eyes on it, soon 👀

File: content/search-github/github-code-search/using-github-code-search.md
Comment (sabrowning1) [inline content/search-github/github-code-search/using-github-code-search.md:44]:
```suggestion
1. Under the search bar, in the "Saved queries" section, click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.
```

Small change to make this flow a little better and combine the two sentences.

File: content/search-github/github-code-search/using-github-code-search.md
Comment (sabrowning1) [inline content/search-github/github-code-search/using-github-code-search.md:46]:
```suggestion
1. In the pop-up window, type both the name you want for your saved search and the query you want to save.
1. To finish creating your saved search, click **Create saved search**.
```

Small change to some pre-existing text to better align with the [suggested terms](https://github.com/github/docs-internal/blob/main/contributing/content-style-guide.md#terms-to-use-or-avoid) in our style guide, and to split these actions into separate steps. Nothing to do with your edits! 😁

File: content/search-github/github-code-search/using-github-code-search.md
Comment (sabrowning1) [inline content/search-github/github-code-search/using-github-code-search.md:48]:
```suggestion
1. To see your saved search, click the search bar. Your saved search will be in the "Saved queries" section. Clicking on a saved search entry will add the query to the search bar and filter the suggestions accordingly.
```

Just a couple more small changes to align this text with the [ordering recommended for procedural steps](https://github.com/github/docs-internal/blob/main/contributing/content-style-guide.md#procedural-steps).

File: content/search-github/github-code-search/using-github-code-search.md
Comment (sabrowning1) [inline content/search-github/github-code-search/using-github-code-search.md:50]:
```suggestion
1. To manage a saved search, type `saved:` in the search bar, then click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.
    To edit a saved search, to the right of the search, click {% octicon "pencil" aria-label="The pencil icon" %}.
    To delete a saved search, to the right of the search, click {% octicon "trash" aria-label="The trash icon" %}.
```

A few more changes to break these steps up and follow the order for procedural steps described in the [style guide](https://github.com/github/docs-internal/blob/main/contributing/content-style-guide.md#procedural-steps) 🙂

**Review by sabrowning1 — COMMENTED** [review]
👋🏼 @npiyush97, thanks so much for opening this PR and linking it to the relevant issue! I've left some comments below with a few changes we should make.

I also really appreciate your extra effort to take those screenshots! Unfortunately, I think removing them makes the most sense for our users. The buttons and icons described in this procedure should be relatively easy to find without visual aids, and adding these screenshots would create additional content for us to maintain and update each time the search function changes. @cmwilson21 is totally right—if you want to learn more about the way we use screenshots in our docs, "[Creating screenshots](https://docs.github.com/en/contributing/writing-for-github-docs/creating-screenshots)" is a great resource. I've left suggestions to remove the screenshot syntax in the Markdown files, and if you could remove the screenshot files themselves, that would be great!

Feel free to ping me when you've made those changes, and I'll take another look 🙂 thanks again for your help maintaining our docs! ✨

**Comment by npiyush97** [comment]
image

@sabrowning1

File: content/search-github/github-code-search/using-github-code-search.md
Comment (sabrowning1) [inline content/search-github/github-code-search/using-github-code-search.md:54]:
```suggestion
1. Under the search bar, in the "Saved queries" section, click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.
1. In the pop-up window, type both the name you want for your saved search and the query you want to save.
1. To finish creating your saved search, click **Create saved search**.
1. To see your saved search, click the search bar. Your saved search will be in the "Saved queries" section. Clicking on a saved search entry will add the query to the search bar and filter the suggestions accordingly.
1. To manage a saved search, type `saved:` in the search bar, then click {% octicon "plus-circle" aria-hidden="true" %} **Manage saved searches**.
    - To edit a saved search, to the right of the search, click {% octicon "pencil" aria-label="The pencil icon" %}.
    - To delete a saved search, to the right of the search, click {% octicon "trash" aria-label="The trash icon" %}.
```

Thanks for making those changes @npiyush97! I'll apply a couple final tweaks:

- we can remove those blank lines between the numbered steps.
- I added dashes to the substeps for managing saved searches to create an indented bulleted list. My apologies for missing that earlier!

I'll go ahead and commit these changes, then get this merged 🙂 thanks again for your help maintaining our docs! ✨

**Review by sabrowning1 — DISMISSED** [review]

**Review by sabrowning1 — APPROVED** [review]

**Pull request merged**
